snippets/get_transaction.py

```diff
@@ -0,0 +1,49 @@
+"""A snippet that walks us through getting a transaction."""
+from xrpl.asyncio.clients import AsyncWebsocketClient
+from xrpl.models.requests import Ledger, Tx
+
+
+async def async_get_transaction() -> None:
+    """
+    Async snippet that walks us through getting a transaction.
+
+    Args:
+        client: The async network client to use to send the request.
+
+    Raises:
+        Exception: if meta not included in the transaction response.
+    """
+    async with AsyncWebsocketClient("wss://s.altnet.rippletest.net:51233") as client:
+        await client.open()
```

Comment:
You don't need to open and close the client when you use `with ... as ...` syntax - that does it automatically for you.